### DISCUSSION TITLE
New values for the category_uid and class_uid of the base_event

### DISCUSSION THREAD

**Discussion by rroupski** [ocsf/ocsf-schema#476] [Proposals]
The change of the `Other` enum value from -1 to 99 affected the base event's `category_uid` and `class_uid`, both are now 99.

<img width="1146" alt="image" src="https://user-images.githubusercontent.com/2128145/217386844-2ab7d34d-e27a-4f39-98b3-b26fbd9a2f13.png">

This is a proposal to change the base event's `category_uid` and `class_uid` values to 0.

<img width="1146" alt="image" src="https://user-images.githubusercontent.com/2128145/217387825-0cb1331e-0fa7-45d4-a5cf-eb9657821ef8.png">

**Comment by rroupski** [comment]
The votes ends EOD Thursday, Feb 9th 2023.

**Comment by floydtree** [comment]
Voting results are in favor of this proposal but is pending implementation.
